nmrfx-processor-gui/src/main/java/org/nmrfx/processor/gui/ChartProcessor.java

```diff
@@ -575,7 +575,7 @@ public Map<String, List<String>> getScriptList() {
         if (mapOpLists != null) {
             for (Map.Entry<String, List<String>> entry : mapOpLists.entrySet()) {
                 List<String> newList = new ArrayList<>();
-                if (entry != null) {
+                if (entry.getValue() != null) {
                     newList.addAll(entry.getValue());
                 }
```

Comment:
I think in this case entry object itself should never be null?, but its value could be so its better to check that instead